### PR TITLE
DBSCAN - Base

### PR DESCRIPTION
**Pull request by aanchalsethi**

### CODE REVIEW

**Comment by ashwani-rathee** [comment]
@aanchalsethi You would need to give a class type implementation of algorithms so that it can be used later.#64 #11 #44 are good examples of proper implementation by your fellow participants.Also add code documentation(in .py file itself) of example on how to test it.

**Comment by aanchalsethi** [comment]
okay, sir will update the file accordingly soon.

**Comment by aanchalsethi** [comment]
Sorry sir I had an exam so i was busy with it. i will update that soon.

**Comment by ashwani-rathee** [comment]
No updates

**Comment by ashwani-rathee** [comment]
This still doesn't have class-type implementation @aanchalsethi Please update

File: DBSCAN/dbscan.py [hunk 1]
```diff
@@ -9,68 +9,19 @@
 #using sklearn
 from sklearn.cluster import DBSCAN
```
Comment (geekquad) [inline DBSCAN/dbscan.py:10]: Omit this. ```sklearn``` not allowed.

**Comment by ashwani-rathee** [comment]
No progress on this

**Comment by aanchalsethi** [comment]
@geekquad Kindly review this file and let me know if there are more changes required.

**Pull request merged**